src/services/lessons/hooks/setCourseId.js

```diff
@@ -0,0 +1,16 @@
+const { BadRequest } = require('@feathersjs/errors');
+
+const setCourseId = (context) => {
+	const { courseId } = context.params.route || {};
+	if (!courseId) {
+		throw new BadRequest('No lessonId or courseId exist.');
+	}
+	if (!context.data) {
```

Comment:
Muss dass immer gesetzt sein? Eventuell lieber default mäßig setzen?

Reply:
Alternativ wäre ist create, patch, oder update. Das ist zwar nur in dem Fall das ein Entwickler den hook falsch verwendet, aber wenn hooks nur für bestimmte Methoden funktionieren sollten sie abgesichert sein. 
Alternativ wäre es möglich das in die Hook Beschreibung mit aufzunehmen.

Reply:
Wenn es nur gegen eine falschverwendung absichern soll, bin ich für in die Beschreibung.